### PR TITLE
chore: OpenTelemetry and Grafana guide

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

I'm still improving the dashboard but the guide is ready for review I think

### CODE REVIEW

File: website/pages/docs/advanced-topics/monitoring/otel-grafana.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    isDefault: true
+    version: 1
+    editable: true
+    apiVersion: 1
+    uid: tempo
+```
+
+This configuration will tell [Grafana][Grafana] to use [Prometheus][Prometheus], [Loki][Loki], and [Tempo][Tempo] as data sources.
+
+## Step 7: Download the CloudQuery Grafana Dashboard
```
Comment (marianogappa) [inline website/pages/docs/advanced-topics/monitoring/otel-grafana.md:189]: Step 6 is missing

File: website/pages/docs/advanced-topics/monitoring/otel-grafana.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    volumes:
+      - tempo_data:/tmp
+      - ./tempo/tempo.yaml:/etc/tempo.yaml
+    ports:
+      - "3200"
+      - "4318"
+  loki:
+    image: grafana/loki:latest
+    ports:
+      - "3100:3100"
```
Comment (marianogappa) [inline website/pages/docs/advanced-topics/monitoring/otel-grafana.md:32]: Is there a meaning to sometimes using the explicit mapping of identical ports, and sometimes doing it implicitly?

File: website/pages/docs/advanced-topics/monitoring/otel-grafana.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+```yaml
+global:
+  scrape_interval: 15s
+scrape_configs:
+  - job_name: "opentelemetry"
+    static_configs:
+      - targets: ["collector:8090"]
+```
+
+This configuration will tell [Prometheus][Prometheus] to scrape the [OpenTelemetry][OTEL] collector every 15 seconds.
```
Comment (marianogappa) [inline website/pages/docs/advanced-topics/monitoring/otel-grafana.md:86]: The `[OpenTelemetry][OTEL]` Markdown bit (btw I don't recognise this syntax) is not working in the UI.

File: website/pages/docs/advanced-topics/monitoring/otel-grafana.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+storage:
+  trace:
+    backend: local
+    wal:
+      path: /tmp/tempo/wal
+    local:
+      path: /tmp/tempo/blocks
+```
+
+This configuration will tell [Tempo][Tempo] to listen on port 3200 and receive [OpenTelemetry][OTEL] traces via HTTP on the default port of 4318.
```
Comment (marianogappa) [inline website/pages/docs/advanced-topics/monitoring/otel-grafana.md:109]: The `[OpenTelemetry][OTEL]` Markdown bit (btw I don't recognise this syntax) is not working in the UI.

**Review by marianogappa — COMMENTED** [review]
As a general knee-jerk reaction to reading the article, there's a lot of steps & moving parts. Is there anything that could be made optional? Do I have to have traces/logs if I just care to know how long my syncs are taking per/table?

File: website/pages/docs/advanced-topics/monitoring/otel-grafana.md (hunk 3 above)
Comment (erezrokah) [reply website/pages/docs/advanced-topics/monitoring/otel-grafana.md]: Fixed in https://github.com/cloudquery/cloudquery/pull/18971/commits/08929c0f3d55ce45e52908e4aa5aada411e38511, learned this syntax from the review I got for https://github.com/DataDog/integrations-extras/pull/2451

**Review by marianogappa — APPROVED** [review]

File: website/pages/docs/advanced-topics/monitoring/otel-grafana.md (hunk 2 above)
Comment (erezrokah) [reply website/pages/docs/advanced-topics/monitoring/otel-grafana.md:32]: So the mapping of identical ports so they can be used from the host. We only need to expose Grafana's `3000` and the collector's `4318` to the host so I'll clean this up.

**Comment by erezrokah** [comment]
> As a general knee-jerk reaction to reading the article, there's a lot of steps & moving parts. Is there anything that could be made optional? Do I have to have traces/logs if I just care to know how long my syncs are taking per/table?

So the dashboard uses traces, metrics and logs for the visualizations. I think one could remove what they want and use https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/nopreceiver or https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/nopexporter in the collector as there's no way to disable individual OTEL features in the CLI. 
That's another piece of feedback from https://github.com/cloudquery/cloudquery-issues/issues/2284.
I'll start by documenting on how to use `noop` in a separate PR.

**Pull request merged**